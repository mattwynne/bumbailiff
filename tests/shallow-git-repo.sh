#!/bin/bash
# shellcheck source=./test_helpers.sh
source "../../../test_helpers.sh"
export GIT_AUTHOR_DATE="$(last_week)"
export GIT_COMMITTER_DATE="$(last_week)"
mkdir -p "origin" && cd "origin" || exit
create_git_repo
git commit --quiet -m "Initial commit" --allow-empty
echo "TODO: fix me" >a-file
git add .
git commit --quiet -m "Add a-file"
touch another-file
git add .
git commit --quiet -m "Add another-file"
cd ..
git clone --quiet "file://$(realpath ./origin)" --depth 1 clone
cd clone || exit
bumbailiff
echo $?
