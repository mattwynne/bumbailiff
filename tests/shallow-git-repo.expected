TODOs could not be scanned: this appears to be a shallow clone of your git repository. Make sure you clone with full depth.
1
